cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -98,8 +96,13 @@ func (p *BicepProvider) Initialize(ctx context.Context, projectPath string, opti
 	if err := tools.EnsureInstalled(ctx, requiredTools...); err != nil {
 		return err
 	}
-
-	return p.EnsureEnv(ctx)
+	if options.BicepADS {
+		p.ignoreADS = options.BicepADS
+	}
+	p.console.ShowSpinner(ctx, "Initialize bicep provider", input.Step)
+	err := p.EnsureEnv(ctx)
+	p.console.StopSpinner(ctx, "", input.Step)
```

Comment:
Why do we need another spinner here?

Reply:
There're like 2 seconds when the console is frozen b/c bicep_provider is initializing.

I'm trying to hide that freezing console with this spinner